lib/arrays/implementations/erlang_array.ex

```diff
@@ -96,27 +90,40 @@ defmodule Arrays.Implementations.ErlangArray do
   defimpl Arrays.Protocol do
     alias Arrays.Implementations.ErlangArray
 
+    @impl true
+    # {:default, val} and {:size, num} are forwarded to `:array`
+    def empty(options) do
```

Comment:
And obviously this one should also be callable explicitly, to remain backwards-compatible.